tests/custom_scalars/test_money.py

```diff
@@ -736,6 +758,32 @@ def test_code():
     await run_sync_test(event_loop, server, test_code)
 
 
+@pytest.mark.asyncio
+@pytest.mark.requests
+async def test_custom_scalar_serialize_variables_sync_transport_2(
+    event_loop, aiohttp_server, run_sync_test
+):
+
+    server, transport = await make_sync_money_transport(aiohttp_server)
+
+    def test_code():
+        with Client(schema=schema, transport=transport, parse_results=True) as session:
+
+            query = gql("query myquery($money: Money) {toEuros(money: $money)}")
+
+            variable_values = {"money": Money(10, "DM")}
+
+            results = session.execute_batch(
+                [GraphQLRequest(document=query, variable_values=variable_values)],
```

Comment:
Maybe we could execute two requests (the same one twice) in the batch here to show that batch is working with multiple queries.

Reply:
ok, seems fair